instrumentation/rxjava-2.0/testing/src/main/groovy/io/opentelemetry/instrumentation/rxjava2/AbstractRxJava2SubscriptionTest.groovy

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.rxjava2
+
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.basicSpan
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.runUnderTrace
+
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.instrumentation.test.InstrumentationSpecification
+import io.reactivex.Single
+import io.reactivex.functions.Consumer
+
+import java.util.concurrent.CountDownLatch
+
+abstract class AbstractRxJava2SubscriptionTest extends InstrumentationSpecification {
+
+  def "subscription test"() {
```

Comment:
Isn't this covered by `AbstractRxJava2Test` already?

Reply:
It was useful when working on this feature. I think it makes no harm to have one more unit test.